### ISSUE TITLE
Commercial licence turn.js

### ISSUE THREAD

**Issue by archimedict** [blasten/turn.js#491]
Dear Emanuel,

We are a Dutch ICT company and interested in using the turn.js api for one of our projects.
The purpose of the finished project is to  build unlimited flip books,  for commercial use, using the turn.js api.

Would you be so kind to inform us about the licence conditions and costs?

Thanks in advance.

Kind regards,

Archimedict BV
Ben ten Holder

**Comment by GoldenCrow** [comment]
I am also interested in a turn js commercial licence. Found this but still awaiting an answer to a few questions myself. 

http://www.turnjs.com/get

Hope this helps

Kind regards,
Peter

**Comment by archimedict** [comment]
Thanks GoldenCrow for your help

**Comment by Noodles947** [comment]
Have either of you managed to get any answers? 

We're also looking at some commercial licenses but we've received no information back from either of the support emails we've sent to Blasten.

Regards,
Andrew Rourke

**Comment by archimedict** [comment]
Dear Andrew,

No response on  our mails to blasten@gmail.com and support@turnjs.com 

Also no response to our ticket on: https://github.com/blasten/turn.js/issues/491#comments 

Met vriendelijke groet

Ben ten Holder

 http://www.archimedict.nl/ www.archimedict.nl

 mailto:bentenholder@archimedict.nl bentenholder@archimedict.nl

cid:image001.jpg@01CAAFB6.01A57610

B.A.G.M. ten Holder
Projectmanager

Randstad 22-50

1316 BZ Almere
Postbus 1151
1300 BD Almere
Nederland
Tel.   +31 36 711 5106

Mob. +31 6 190 19556
Mob. +31 6 519 11 601

Van: Noodles947 [mailto:notifications@github.com] 
Verzonden: dinsdag 25 november 2014 10:30
Aan: blasten/turn.js
CC: archimedict
Onderwerp: Re: [turn.js] Commercial licence turn.js (#491)

Have either of you managed to get any answers? 

We're also looking at some commercial licenses but we've received no information back from either of the support emails we've sent to Blasten.

Regards,
Andrew Rourke

—
Reply to this email directly or view it on GitHub https://github.com/blasten/turn.js/issues/491#issuecomment-64331344 .  https://github.com/notifications/beacon/AJWCeienE-jKrd6Ohw_Ewp6rP6-xws1Eks5nREOagaJpZM4C8YDd.gif 

Geen virus gevonden in dit bericht.
Gecontroleerd door AVG - www.avg.com
Versie: 2015.0.5577 / Virusdatabase: 4223/8627 - datum van uitgifte: 11/25/14

**Comment by hnuecke** [comment]
I'm waiting for about 2 years now for an announced / promised update. 
The last email I received from him is almost from 2 years ago. I sent another one 15 months ago without getting an answer. I tried again half a year ago; no answer neither. He's active on twitter; but talks about everything but turnjs.
My feeling is he gave up that project; for whatever reason!
Too bad, since the general architecture and performance is nice!
I planned to use turnjs 2 years ago; but only after a next release (expecting improvements regarding zoom support and more). 
The situation as very disappointing and a waste of money (I paid for a commercial license but never used it).

**Comment by stefek99** [comment]
+1 

I might be interested in obtaining the licence for a client project...

(sorry to hear that you have troubles with contact)

**Comment by yhakimi** [comment]
Same issue, No response to our emails to the address mentioned on the website

**Comment by act2281** [comment]
This page seems to be very usable: http://www.turnjs.com/get

**Comment by joshuajonah** [comment]
Yep, they use it to accept your money then never cobtact you back.
